entrypoint.sh

```diff
@@ -171,6 +172,15 @@ function start_postfix {
     echo "$HOSTNAME" > /etc/mailname
     echo "$HOSTNAME" > /etc/hostname
 
+    if [ "$SMF_SMARTHOST" = "" ]; then
+        SMF_SMARTHOST=$1
```

Comment:
I think it's not necessary to use `$1` as the default value of `SMF_RELAYHOST`. Because it's not a so important setting for all users.

If there's no SMF_RELAYHOST set, we should just ignore it.